Source/WTF/wtf/text/StringBuilder.h

```diff
@@ -60,10 +60,10 @@ class StringBuilder {
     void append(const String&);
     void append(StringView);
     void append(ASCIILiteral);
+    void append(const char*) = delete; // Pass ASCIILiteral or span instead.
```

Comment:
You’ll want to do the same kind of thing in `StringConcatenate.h` eventually, for the same reason. After that, you might be able to remove this explicit `delete`.

Reply:
Started work on this: https://github.com/WebKit/WebKit/pull/28161